plugins/networktables/src/main/java/edu/wpi/first/shuffleboard/plugin/networktables/TabGenerator.java

```diff
@@ -125,6 +127,16 @@ private void metadataChanged(EntryNotification event) {
       }
     }
 
+    // Selected tab
+    if (name.endsWith("/" + SELECTED_ENTRY_NAME)) {
```

Comment:
PMD is flagging this class as a possible god class (basically, there's too much "stuff" going on). Try moving this code to its own method; it that doesn't work, add `@SuppressWarnings("PMD.GodClass")` to the class declaration and I'll clean it up later